README.md

```diff
@@ -408,6 +408,29 @@ Example:
       thing'
 ```
 
+### Action: `append`
+
+Appends a string on the end of a given file. File must already exist. If no
+newline at tend of `with` parameter, one will be added unless
+`skip-ensure-newline` is set to `true`.
+
+Params:
+- `path`: File in which to do the replacement.
```

Comment:
All the other actions take `paths` (plural) rather than `path`, should we keep that for consistency? Maybe there's not much legitimate need to append to multiple files? I'm on the fence.

Reply:
That's a good question, I don't have a strong opinion.

`path` is simpler and someone could use `regex_replace` if they really needed paths, but I do like having uniformity in APIs

Reply:
Lets go with `paths` for uniformity. It's easy to wrap a single path with `[` `]`.

Reply:
Changed and tests added